app/models/vm_reconfigure_task.rb

```diff
@@ -30,7 +30,7 @@ def self.get_description(req_obj)
     new_settings << "Processor Sockets: #{req_obj.options[:number_of_sockets].to_i}" if req_obj.options[:number_of_sockets].present?
     new_settings << "Processor Cores Per Socket: #{req_obj.options[:cores_per_socket].to_i}" if req_obj.options[:cores_per_socket].present?
     new_settings << "Total Processors: #{req_obj.options[:number_of_cpus].to_i}" if req_obj.options[:number_of_cpus].present?
-    new_settings << "Add Disks: #{req_obj.options[:disk_add].length}" if req_obj.options[:disk_add].present?
+    new_settings << "Add Disks: #{req_obj.options[:disk_add].length} : #{req_obj.options[:disk_add].collect { |d| d["disk_size_in_mb"].to_i.megabytes.to_s(:human_size) }.join(", ")} " if req_obj.options[:disk_add].present?
```

Comment:
Overall the change looks good and since this will be back-ported I think it is ok as is.

@billfitzgerald0120 I would like to spend a little time with you on refactoring this class as the majority of the logic is 
`new_setting << <some string> if <some key>.present?`

I think we could DRY up this logic and make the code easier to read.